pkg/server/server_sql.go

```diff
@@ -770,7 +770,7 @@ func (s *sqlServer) preStart(
 		return errors.Wrap(err, "ensuring SQL migrations")
 	}
 
-	log.Infof(ctx, "done ensuring all necessary migrations have run")
+	log.Infof(ctx, "done ensuring all necessary startup migrations have run")
```

Comment:
:+1: I'd say throw this one into a separate PR, as this one won't merge for a while and that way it's out of your hair

Reply:
https://github.com/cockroachdb/cockroach/pull/56194 (I never know with small diffs like this, they're not cumbersome enough for me to want to pull them out separately).